package.json

```diff
@@ -9,7 +9,7 @@
     ]
   },
   "dependencies": {
-    "@nyaruka/flow-editor": "1.6.2",
+    "@greatnonprofits-nfp/flow-editor": "greatnonprofits-nfp/floweditor#ae75c11979fd3de5e08f4253cba0d50679266435",
```

Comment:
I tried to run the `npm install` or `yarn install`, but the library was installed without the 
compiling process

<img width="255" alt="Screen Shot 2019-11-26 at 19 09 39" src="https://user-images.githubusercontent.com/5198870/69676817-53711680-1080-11ea-9f89-af69dac78ac1.png">

Reply:
I had to manually build it might collide with the changes I made on floweditor for the feature of less than one minute. But now I have pushed an update.